README.md

```diff
@@ -26,14 +26,24 @@ Demonstrate your understanding of this Sprint's concepts by answering the follow
 
 - [ ] Mention two parts of Express that you learned about this week.
 
+    req and res
+
 - [ ] Describe Middleware?
 
+    middleware is a function or functions hat get executed in the order they are introduced into the server code.
+
 - [ ] Describe a Resource?
 
+    A resource a route to a database.
```

Comment:
grouping / category in a database. (Users / locations / sneakers etc etc depending on the purpose of the app)